js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -193,33 +194,37 @@ export default function SetupPaidAds() {
 			{ showPaidAdsSetup && (
 				<PaidAdsSetupSections onStatesReceived={ setPaidAds } />
 			) }
-			<FaqsSection />
-			<StepContentFooter hidden={ ! showPaidAdsSetup }>
-				<Flex justify="right" gap={ 4 }>
-					{ createSkipButton(
-						__(
-							'Skip paid ads creation',
-							'google-listings-and-ads'
-						)
-					) }
-					<AppButton
-						isPrimary
-						data-action={ ACTION_COMPLETE }
-						text={ __(
-							'Complete setup',
-							'google-listings-and-ads'
+
+			<StepContentFooter>
+				<StepContentActions hidden={ ! showPaidAdsSetup }>
+					<Flex justify="right" gap={ 4 }>
+						{ createSkipButton(
+							__(
+								'Skip paid ads creation',
+								'google-listings-and-ads'
+							)
 						) }
-						loading={ completing === ACTION_COMPLETE }
-						disabled={ disabledComplete }
-						onClick={ handleCompleteClick }
-						eventName="gla_onboarding_complete_with_paid_ads_button_click"
-						eventProps={ {
-							budget: paidAds.amount,
-							audiences: paidAds.countryCodes?.join( ',' ),
-						} }
-					/>
-				</Flex>
+						<AppButton
+							isPrimary
+							data-action={ ACTION_COMPLETE }
+							text={ __(
+								'Complete setup',
+								'google-listings-and-ads'
+							) }
+							loading={ completing === ACTION_COMPLETE }
+							disabled={ disabledComplete }
+							onClick={ handleCompleteClick }
+							eventName="gla_onboarding_complete_with_paid_ads_button_click"
+							eventProps={ {
+								budget: paidAds.amount,
+								audiences: paidAds.countryCodes?.join( ',' ),
+							} }
+						/>
+					</Flex>
+				</StepContentActions>
 			</StepContentFooter>
 		</StepContent>
 	);
 }
+
+<PaidAdsFaqsPanel />;
```

Comment:
It looks like this should be placed inside  `StepContentFooter` component, otherwise the FAQ will be not displayed.

Reply:
Thanks for catching that @jorgemd24 . PR Updated!